### ISSUE TITLE
[Question] Remove `Auto reschedule cards reviewed on other devices after sync`

### ISSUE THREAD

**Issue by KAGEYAM4** [open-spaced-repetition/fsrs4anki-helper#479]
**Research**

*Enter an [x] character to confirm the points below:*

- [x] I have checked the [FAQ](https://github.com/open-spaced-repetition/fsrs4anki/wiki/FAQ) and could not find an answer to my question
- [x] I have read the [wiki](https://github.com/open-spaced-repetition/fsrs4anki/wiki) and still felt confused
- [x] I have searched for similar existing questions here

**Question**
![image](https://github.com/user-attachments/assets/06e09de0-fbaa-4096-85b6-0de85aabf4f0)

Will the option `Auto reschedule cards reviewed on other devices after sync` going to be removed now that every platform supports FSRS?

**Comment by L-M-Sherlock** [comment]
@user1823, @Expertium, should we remove this option?

**Comment by Expertium** [comment]
It depends on how many people are still using old versions of Anki that don't support FSRS. Probably not that many.
But then again, this option doesn't hurt anybody and removing it wouldn't significantly improve the UI because there are a lof of other options.
So idk

**Comment by user1823** [comment]
I also agree that we don't have a good reason to remove it yet. Plus, this option is also useful for users who have a new FSRS version on desktop and an old version on mobile.

**Comment by L-M-Sherlock** [comment]
OK. Just keep it.
